src/RoutingList/CreateRoutingList/CreateRoutingListFormContainer.js

```diff
@@ -0,0 +1,57 @@
+import noop from 'lodash/noop';
+import { useCallback } from 'react';
+import { FormattedMessage } from 'react-intl';
+import {
+  useHistory,
+  useParams,
+} from 'react-router';
+
+import { useShowCallout } from '@folio/stripes-acq-components';
+
+import { UNIQUE_NAME_ERROR_CODE } from '../constants';
+import { useRoutingListMutation } from '../hooks';
+import RoutingListForm from '../RoutingListForm';
+
+export const CreateRoutingListFormContainer = () => {
+  const showCallout = useShowCallout();
+  const history = useHistory();
+  const { poLineId } = useParams();
+  const { createListing } = useRoutingListMutation();
+
+  const onClose = useCallback(() => {
```

Comment:
what is the purpose of useCallback here when onSubmit is not memoized? decide you need useCallback or not and what you want to achieve with it